src/main/scala/com/fortysevendeg/translatebubble/modules/clipboard/impl/ClipboardServicesComponentImpl.scala

```diff
@@ -61,7 +61,7 @@ trait ClipboardServicesComponentImpl
       } yield text
 
       result match {
-        case Some(text) if text.toString.trim.length > 0 => lastDate = currentMillis; currentInterval > millisInterval
+        case Some(text) if isValidText(text) => lastDate = currentMillis; currentInterval > millisInterval
```

Comment:
Your code is perfect but as I know is you first Scala contribution and I going to be more `tiki-tiki` but it's just for your scala knowledge ;) . Of course, it could be out of the scope of this PR but I hope would be useful.
Instead of dealing with the CharSequence in the private functions, as alternative we could pass as a parameter the string directly.

`case Some(text) if isValidText(text.toString)`